MotMystere.cpp

```diff
@@ -2,19 +2,53 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
 
 int main()
 {
-    std::cout << "Hello World!\n";
+   
+    
+    //1 : On demande de saisir un mot
+
+    string motMystere("Procra");
+    cout << "Entrer un mot : ";
+    std::cin >> motMystere;
+    cout << "Le mot est :" << motMystere;
+    return 0;
```

Comment:
Why return here? If you put your return statement here, you never get to the lines of code below, so your program stops here. Is this the intended behavior?

Reply:
dont know=)but thx, now I understand better I think..